newIDE/app/src/InAppTutorial/SchemaChecker.js

```diff
@@ -30,6 +30,8 @@ const flowStepKeys = [
   'tooltip',
   'skippable',
   'isOnClosableDialog',
+  'interactsWithCanvas',
+  'disableBlockingLayer',
```

Comment:
new prop to completely disable blocking layer on a step, in case it's too hard to create meaningful holes

Reply:
This should be added in the documentation

Reply:
done